java/main/com/google/privacy/differentialprivacy/LaplaceNoise.java

```diff
@@ -149,17 +150,58 @@ public ConfidenceInterval computeConfidenceInterval(
       long lInfSensitivity,
       double epsilon,
       @Nullable Double delta,
-      double confidenceLevel) {
-    // TODO: Implement confidence interval computation.
-    throw new UnsupportedOperationException("Not implemented yet.");
+      double alpha) {
+    checkConfidenceParameters(l0Sensitivity, lInfSensitivity, epsilon, delta, alpha);
+    ConfidenceInterval confInt =
+        computeConfidenceInterval(
+            noisedX, Noise.getL1Sensitivity(l0Sensitivity, lInfSensitivity) / epsilon, alpha);
+    return ConfidenceInterval.create(
+        Math.round(confInt.lowerBound()), Math.round(confInt.upperBound()));
   }
 
   private void checkParameters(double l1Sensitivity, double epsilon, @Nullable Double delta) {
-    DpPreconditions.checkEpsilon(epsilon);
+    DpPreconditions.checkEpsilonVeryStrict(epsilon);
     DpPreconditions.checkNoiseDelta(delta, this);
     DpPreconditions.checkL1Sensitivity(l1Sensitivity);
   }
 
+  private void checkConfidenceParameters(
+      int l0Sensitivity,
+      double lInfSensitivity,
+      double epsilon,
+      @Nullable Double delta,
+      double alpha) {
+    DpPreconditions.checkEpsilonStrict(epsilon);
```

Comment:
I know that epsilon does not have to be very strict for this to work, but since all sorts of stuff may happen for super small epsilon (overflows, arithmetic imprecision, etc.) and we can't compute noise for these parameters anyway (not that there would be a use case for it) we can make our lives easier and just use checkEpsilonVeryStrict(epsilon). In particular, you can reuse checkParameters here.

Reply:
Yeah I agree but this was requested in the Java interface PR and was implemented in Go and merged.
https://github.com/google/differential-privacy/pull/33#discussion_r467125967
Arithmetic imprecision might come from lambda computation and would return infinite bounds in case of an overflow which makes sense if you are using values on that scale. Alternatively, we can change this here and make a PR to Go to change it.